pkgs/development/python-modules/azure-servicemanagement-legacy/default.nix

```diff
@@ -1,31 +1,39 @@
-{ pkgs
+{ lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , azure-common
 , requests
 , python
+, isPy3k
```

Comment:
This doesn't get used in the expression :)

Reply:
XD
aaaaaand that's _exactly_ why I need to automate this biz...

`python` doesn't get used either, now that the `rm` statements are gone.

Just pushed a rebase to fix it. Many thanks! =]